packages/backend/src/index.ts

```diff
@@ -12,6 +12,7 @@ import env from './config/env.js'
 
 import { errorHandler } from './middlewares/errorHandler'
 import { mainRouter } from './routes.js'
+import { getAcceptableIDtypesByCountry } from './rapyd/verification/index.js'
```

Comment:
```suggestion
import { getAcceptableIDtypesByCountry } from './rapyd/verification'
```

There are more imports that specify a `.js` file.